EMF/events/decadence_events.txt

```diff
@@ -269,9 +269,10 @@ character_event = {
 	capable_only = yes
 	prisoner = no
 	only_rulers = yes
+	religion_group = muslim

```

Comment:
You know what's weird about this `religion_group = muslim` pre-trigger?  The trigger itself has been modified to use `uses_decadence = yes`.